fda-model-test/unittests/ImpactAreaScenarioResultsShould.cs

```diff
@@ -13,8 +13,8 @@ namespace fda_model_test.unittests
 {
     [Trait("Category", "Unit")]
 
-    public class ResultsShould
-    {
+    public class ImpactAreaScenarioResultsShould
+    {//TODO: Access these results through ScenarioREsults 
```

Comment:
TODO statement. If you need to leave it, that is fine. Just pointing it out in case you missed it.